tech.json

```diff
@@ -1837,7 +1889,12 @@
                 "canCrystalFlash",
                 "canCeilingClip"
               ],
-              "otherRequires": [],
+              "otherRequires": [
+                {"or": [
+                  "SpringBall",
```

Comment:
Hopefully if you are doing `canJumpIntoCrystalFlashClip`, you understand the nuance of doing it with Springball.

Reply:
You'll still have Morph in the `canCrystalFlash` tech, and presumably can midair morph.

The CF tech and helpers could use a bit more cleanup after this PR as well.